src/topology/vector_bundle.lean

```diff
@@ -387,3 +410,168 @@ lemma is_open_map_proj : is_open_map Z.proj :=
 topological_fiber_bundle_core.is_open_map_proj Z
 
 end topological_vector_bundle_core
+
+section sections
+
+/-! ### Sections of topological vector bundles -/
+
+/-- Type synonim to allow to declare instances that depend on implicit parameters containing `R`
+and `F`. -/
+@[reducible, nolint unused_arguments]
+def topological_vector_bundle.vector_bundle_section (R : Type*) {B : Type*} (F : Type*)
+  (E : B → Type*) [topological_space F] [topological_space (total_space E)] [topological_space B] :=
+continuous_bundle_section E
+
+open topological_vector_bundle
+
+variables {R B E F}
+
+lemma right_inv.image_mem_trivialization_at_source (f : right_inv (proj E)) (b : B) :
+  f b ∈ (trivialization_at R F E b).source :=
+f.mem_base_set_image_mem_source (mem_base_set_trivialization_at R F E b)
+
+variables (R F E)
+
+lemma bundle_section.continuous_at_iff_continuous_within_at_triv_at (f : bundle_section E) (b : B) :
+  continuous_at f b ↔ continuous_within_at (λ x, ((trivialization_at R F E b) (f x)).snd)
+  (trivialization_at R F E b).base_set b :=
+(f : right_inv (proj E)).continuous_at_iff_continuous_within_at (trivialization_at R F E b)
+  (mem_base_set_trivialization_at R F E b)
+
+variables {E}
+
+section
+
+include R F
+
+lemma continuous.add_section [has_continuous_add F] {g h : bundle_section E} (hg : continuous g)
```

Comment:
It would be better to build a proper API here. I.e., prove that if `f` and `g` are continuous at `x` within a set `s`, then so is `f+g`. And then prove versions for `continuous_at`, `continuous_on` and `continuous`. You can look at how the API is built for differentiable functions or times_cont_diff functions.

Reply:
Done